packages/modeling/src/operations/hulls/quickhull/Face.js

```diff
@@ -0,0 +1,327 @@
+const add = require('../../../math/vec3/add')
```

Comment:
Nice work adapting the code of quickhull3D to our stuff !
I think that for each of these quickhull files we should be very clear that they are not based on our code but are a modified versions of https://www.npmjs.com/package/quickhull3d (licence & attribution of code origin are very important :)

Reply:
@kaosat-dev please review the files again. i added copyright and information to each of the files in the quickhull library.

Reply:
@z3dev : thanks for updating !
One last thing is having that information in the README as well and it is all golden ! :)

Reply:
I added copyright information to README.md for the quickhull library.

I was thinking... shouldn't we add a copyright for the gl-matrix library?

Reply:
hmm that is a good tricky question
we are not using the code as such and we have diverged a fair bit, but perhaps it is both more nice, correct and open to clearly put 'the maths library and modularity is based on the gl-matrix and its split up versions'
I will add this to the list of needed tweaks before V2 release, and merge this in the meantime